Packs/CortexAttackSurfaceManagement/pack_metadata.json

```diff
@@ -55,8 +55,9 @@
         "ServiceNow",
         "Nmap",
         "Tenable_io",
+        "Azure-Enrichment-Remediation",
```

Comment:
at line 45 add:
"Azure-Enrichment-Remediation": {
        "mandatory": false,
        "display_name": "Azure Enrichment and Remediation"
    }

Reply:
resolved in https://github.com/demisto/content/pull/24310/commits/c88af851fee1e0dde19ab7d443813494a48c2f27